src/config/site.json

```diff
@@ -2,7 +2,7 @@
   "CR": {
     "title": "Giftaid",
     "title_postfix": " | Comic Relief",
-    "logo": "/images/cr_logo.svg",
+    "logo": "/images/SR2022-logo.png",
```

Comment:
We really wanna be updating the SR site here, rather than the CR site

Reply:
(just realised that both local and the PR env 'default' to being CR, which isn't super helpful here)

Reply:
Obviously don't commit the change, but you could shortcircuit this logic [here](https://github.com/comicrelief/giftaid-react/blob/master/src/service/Site.service.js#L31) in order to force the "use SR config" in your local instance

Reply:
Tried this hack, but it still doesn't show on local:
```  getSite() {
    const hostname = window.location.hostname;

    if(hostname.includes('3000')) {
      return 'SR';
    }

    if(hostname.includes('sportrelief')) {
      return 'SR';
    }

    if(hostname.includes('rednoseday')) {
      return 'RND';
    }

    if(hostname.includes('bignightin')) {
      return 'BIGNIGHTIN';
    }

    return 'CR';
  }
```

Reply:
Yay, this works on local:
   ```
 if(hostname.includes('localhost')) {
      return 'SR';
    }

```

Reply:
> Tried this hack, but it still doesn't show on local:
> 
> ```
>     const hostname = window.location.hostname;
> 
>     if(hostname.includes('3000')) {
>       return 'SR';
>     }
> ...
> ```

That's because `3000` is the _port_ here, not the _domain_; `localhost` is, that's why it worked :)